### PR TITLE
65371030 add a delete action to tube resource

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

File: lib/lims-laboratory-app/laboratory/tube/delete_tube.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      end
+    end
+  end
+
+  module Laboratory
+    class Tube
+      Delete = DeleteTube
+    end
+  end
+end
```
Comment (maxigit) [inline lib/lims-laboratory-app/laboratory/tube/delete_tube.rb:26]: Can't we have a generic DeleteAction in the core which will work for everything instead ?

File: requests/apiary/2_root.json.erb [hunk 2]
```diff
@@ -425,6 +425,11 @@ h[:response]=<<EOD
              "create": "http://example.org/actions/update_tube_rack"
            }
          },
+         "delete_tubes": {
+           "actions": {
+             "create": "http://example.org/actions/delete_tube"
+           }
+         },
```
Comment (maxigit) [inline requests/apiary/2_root.json.erb:432]: Is this url usable ? It seem that the standard url to delete something is 
   DELETE <uuid>

Comment (ke4) [inline requests/apiary/2_root.json.erb:432]: Yes it is usable.
In the lims-api we have the "create" url for everything (create, update, delete).
Please, check the other delete or update actions.

**Comment by ke4** [comment]
I've create a general delete action, so this is not needed now.
Here is the pull request for the general delete action:
https://github.com/sanger/lims-api/pull/213

**Comment by maxigit** [comment]
It's been merged already
Max

On Mar 12, 2014, at 2:23 PM, Károly Erdős notifications@github.com wrote:

> I've create a general delete action, so this is not needed now.
> Here is the pull request for the general delete action:
> sanger/lims-api#213
> 
> —
> Reply to this email directly or view it on GitHub.

## 

 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

**Pull request closed without merging**